device/iot-device-samples/send-serialized-event/src/main/java/samples/com/microsoft/azure/sdk/iot/ContosoAnemometer.java

```diff
@@ -7,7 +7,9 @@
 
 public class ContosoAnemometer {
     public String deviceId;
-    public double windSpeed;
+    public int messageId;
+    public double temperature;
```

Comment:
We use messageId in the packet structure that we use at docs.microsoft.com for all the samples for the physical devices (e.g. https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-arduino-huzzah-esp8266-get-started)